projects/angular/src/layout/nav/aria-current-link.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, HostBinding, OnDestroy, OnInit } from '@angular/core';
+import { RouterLinkActive } from '@angular/router';
+import { Subscription } from 'rxjs';
+
+@Directive({ selector: '[clr-aria-current-link]', host: { '[attr.aria-current]': 'stepAriaCurrent' } })
+export class ClrAriaCurrentLink implements OnInit, OnDestroy {
+  @HostBinding('attr.aria-current') ariaCurrent = undefined;
```

Comment:
You are setting host binding in two different places in two different ways.

```
host: { '[attr.aria-current]': 'stepAriaCurrent'. } 
```

and 

```
@HostBinding('attr.aria-current') ariaCurrent = undefined;
```

The first one is preferred as you don't need to import `HostBinding` decorator.

Reply:
I was trying the first and it didn't seem to be working, so I tried switching to the second and thought I deleted the first, Turns out the issue was that I wasn't subscribing to the activeChanged event. 

Good catch, I'll clean this up